test-framework/clustering/src/main/java/org/keycloak/testframework/clustering/LoadBalancerSupplier.java

```diff
@@ -20,8 +23,23 @@ public LoadBalancer getValue(InstanceContext<LoadBalancer, InjectLoadBalancer> i
         throw new IllegalStateException("Load balancer can only be used with ClusteredKeycloakServer");
     }
 
+    @Override
+    public void close(InstanceContext<LoadBalancer, InjectLoadBalancer> instanceContext) {
+        instanceContext.getValue().close();
+    }
+
     @Override
     public boolean compatible(InstanceContext<LoadBalancer, InjectLoadBalancer> a, RequestedInstance<LoadBalancer, InjectLoadBalancer> b) {
         return true;
     }
+
+    @Override
+    public int order() {
+        return SupplierOrder.BEFORE_REALM;
```

Comment:
@ryanemerson Should it be started after the Keycloak startup?

Reply:
Locally I have it working with `BEFORE_KEYCLOAK_SERVER`, however it requires changes to the LoadBalancer implement as well as the ordering changes to `Registry.java`. I think starting the LoadBalancer after startup and before the realm is fine, as that seems to ensure that all test setup etc will work as expected via the LoadBalancer.